src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
```

Comment:
Don't begin class or method implementation with an empty line. There should be no empty line after the class declaration.